.idea/libraries/Maven__org_projectlombok_lombok_unknown.xml

```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
+  <library name="Maven: org.projectlombok:lombok:unknown">
+    <CLASSES>
+      <root url="jar://$MAVEN_REPOSITORY$/org/projectlombok/lombok/unknown/lombok-unknown.jar!/" />
+    </CLASSES>
+    <JAVADOC>
+      <root url="jar://$MAVEN_REPOSITORY$/org/projectlombok/lombok/unknown/lombok-unknown-javadoc.jar!/" />
+    </JAVADOC>
```

Comment:
123123

Reply:
reply katerina by chenyan, by new review person is qq

Reply:
reply katerina by qq, and new review person is qq